demo: browserify is not working now.

I had followd the step in the README.md.

.... 
D:\TypeScriptSamples\browserify>npm run tsc

> typescript-browserify@1.0.0 tsc D:\TypeScriptSamples\browserify
> node node_modules/typescript/bin/tsc.js

module.js:338
    throw err;
    ^

Error: Cannot find module 'D:\TypeScriptSamples\browserify\node_modules\typescript\bin\tsc.js'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
    at node.js:951:3

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "tsc"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE
npm ERR! typescript-browserify@1.0.0 tsc: `node node_modules/typescript/bin/tsc.js`
npm ERR! Exit status 1

--- a/browserify/package.json
+++ b/browserify/package.json
@@ -13,7 +13,7 @@
         "typescript": "^1.5.3"
     },
     "scripts": {
-        "tsc": "node node_modules/typescript/bin/tsc.js",
+        "tsc": "node node_modules/typescript/lib/tsc.js",
         "browserify": "browserify src/app.js -o bundle.js -s app",
         "listen": "node node_modules/http-server/bin/http-server",
         "all": "npm run tsc &&  npm run browserify && npm run listen"

--- a/browserify/package.json
+++ b/browserify/package.json
@@ -10,7 +10,7 @@
         "http-server": "0.8.0"
     },
     "devDependencies": {
-        "typescript": "^1.5.3"
+        "typescript": "^1.8.9"
     },
     "scripts": {
         "tsc": "node node_modules/typescript/lib/tsc.js",

